gcloud/bigquery/table.py

```diff
@@ -419,6 +419,8 @@ def _build_resource(self):
         if self.view_query is not None:
             view = resource['view'] = {}
             view['query'] = self.view_query
+            # https://github.com/GoogleCloudPlatform/gcloud-python/issues/1701
```

Comment:
The comment explains why one is mutating an apparently-unrleated bit of the resource when passing the `{'view': {'query': QUERY_STRING}}` element.